default_joints:
  - index: 0
    name: left_shoulder_pitch_joint
    pos_des: 0.0
    vel_des: 0.0
    kp: 100.0
    kd: 10.0
    torque: 5.0
  - index: 1
    name: left_shoulder_roll_joint
    pos_des: 0.5
    vel_des: 0.1
    kp: 150.0
    kd: 15.0
    torque: 6.0
  - index: 2
    name: left_arm_yaw_joint
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
  - index: 3
    name: left_elbow_pitch_joint
    pos_des: 0.0
    vel_des: 0.0
    kp: 100.0
    kd: 10.0
    torque: 5.0
  - index: 4
    name: left_elbow_yaw_joint
    pos_des: 0.5
    vel_des: 0.1
    kp: 150.0
    kd: 15.0
    torque: 6.0
  - index: 5
    name: left_wrist_roll_joint # LEFT_WRIST_ROLL_JOINT
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
  - index: 6
    name: left_wrist_yaw_joint # LEFT_WRIST_YAW_JOINT
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
  - index: 7
    name: neck_yaw_joint # NECK_YAW_JOINT
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
  - index: 8
    name: neck_pitch_joint # NECK_PITCH_JOINT
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
  - index: 9
    name: right_shoulder_pitch_joint
    pos_des: 0.0
    vel_des: 0.0
    kp: 100.0
    kd: 10.0
    torque: 5.0
  - index: 10
    name: right_shoulder_roll_joint
    pos_des: 0.5
    vel_des: 0.1
    kp: 150.0
    kd: 15.0
    torque: 6.0
  - index: 11
    name: right_arm_yaw_joint
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
  - index: 12
    name: right_elbow_pitch_joint
    pos_des: 0.0
    vel_des: 0.0
    kp: 100.0
    kd: 10.0
    torque: 5.0
  - index: 13
    name: right_elbow_yaw_joint
    pos_des: 0.5
    vel_des: 0.1
    kp: 150.0
    kd: 15.0
    torque: 6.0
  - index: 14
    name: right_wrist_roll_joint # RIGHT_WRIST_ROLL_JOINT
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
  - index: 15
    name: right_wrist_yaw_joint # RIGHT_WRIST_YAW_JOINT
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
  - index: 16
    name: waist_yaw_joint # WAIST_YAW_JOINT
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
  - index: 17
    name: waist_roll_joint # WAIST_ROLL_JOINT
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
  - index: 18
    name: left_leg_roll_joint # LEFT_LEG_ROLL_JOINT
    pos_des: 0.0
    vel_des: 0.0
    kp: 100.0
    kd: 10.0
    torque: 5.0
  - index: 19
    name: left_leg_yaw_joint # LEFT_LEG_YAW_JOINT
    pos_des: 0.5
    vel_des: 0.1
    kp: 150.0
    kd: 15.0
    torque: 6.0
  - index: 20
    name: left_leg_pitch_joint # LEFT_LEG_PITCH_JOINT
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
  - index: 21
    name: left_knee_joint # LEFT_KNEE_JOINT
    pos_des: 0.0
    vel_des: 0.0
    kp: 100.0
    kd: 10.0
    torque: 5.0
  - index: 22
    name: left_ankle_pitch_joint # LEFT_ANKLE_PITCH_JOINT
    pos_des: 0.5
    vel_des: 0.1
    kp: 150.0
    kd: 15.0
    torque: 6.0
  - index: 23
    name: left_ankle_roll_joint # LEFT_ANKLE_ROLL_JOINT
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
  - index: 24
    name: right_leg_roll_joint # RIGHT_LEG_ROLL_JOINT
    pos_des: 0.0
    vel_des: 0.0
    kp: 100.0
    kd: 10.0
    torque: 5.0
  - index: 25
    name: right_leg_yaw_joint # RIGHT_LEG_YAW_JOINT
    pos_des: 0.5
    vel_des: 0.1
    kp: 150.0
    kd: 15.0
    torque: 6.0
  - index: 26
    name: right_leg_pitch_joint # RIGHT_LEG_PITCH_JOINT
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
  - index: 27
    name: right_knee_joint # RIGHT_KNEE_JOINT
    pos_des: 0.0
    vel_des: 0.0
    kp: 100.0
    kd: 10.0
    torque: 5.0
  - index: 28
    name: right_ankle_pitch_joint # RIGHT_ANKLE_PITCH_JOINT
    pos_des: 0.5
    vel_des: 0.1
    kp: 150.0
    kd: 15.0
    torque: 6.0
  - index: 29
    name: right_ankle_roll_joint # RIGHT_ANKLE_ROLL_JOINT
    pos_des: -0.5
    vel_des: -0.1
    kp: 200.0
    kd: 20.0
    torque: 7.0
